platform/android/dependencies.gradle

```diff
@@ -7,21 +7,21 @@ ext {
     versionCode = 11
     versionName = "5.0.0"
 
-    supportLibVersion = "25.3.1"
-    leakCanaryVersion = '1.5'
+    mapboxServicesVersion = "2.2.0-SNAPSHOT"
+    supportLibVersion = "25.1.0"
```

Comment:
at some point yes :smiley: I would aim at coordinating this with a final release? (would also like to ping LOST for this, they have a really outdated one).